GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -98,7 +98,6 @@ public CommitInfo()
                 this.InvokeAsync(() =>
                 {
                     UICommandsSource.UICommandsChanged += UICommandsSource_UICommandsChanged;
-                    RefreshSortedRefs();
```

Comment:
Is necessary for "Contained in branches/tags". Restored in #6727.

Reply:
Thank you, I couldn't quite work out what this was for...